axelrod/random_.py

```diff
@@ -3,7 +3,7 @@
 from axelrod import Actions
 
 
-def random_choice(p=0.5):
+def random_choice(p=0.5: float):
```

Comment:
@Nikoleta-v3 Nice to meet you! 
I will look into the review.

@marcharper What should be the return type set? Should it be imported as Action.